crates/syntax/src/ast/make.rs

```diff
@@ -158,6 +160,52 @@ fn ty_from_text(text: &str) -> ast::Type {
     ast_from_text(&format!("type _T = {text};"))
 }
 
+/** Related goto [link](https://doc.rust-lang.org/reference/items/type-aliases.html)
```

Comment:
We don't really use block comments in the codebase (I think there might be an assist that swithces between the two even)